README.md

```diff
@@ -9,58 +9,114 @@ Diagnostic and Debugging Tools for Ditto in Flutter
 
 // In your widget build method
 Scaffold(
-  appBar: AppBar(title: Text('Permissions Health')),
-  body: PermissionsHealthView(),
+  appBar: AppBar(title: Text('Connected Peers')),
+  body: PeerListView(ditto: myDittoInstance),
 )
 ```
 
 ### Features
 
-The permissions health view monitors the following:
+The peer list view provides:
 
-1. **Bluetooth Permission** - Shows whether your app has been granted Bluetooth access
-2. **Bluetooth Status** - Shows if Bluetooth is enabled/disabled on the device
-3. **Wi-Fi Status** - Shows peer-to-peer WiFi capabilities (WiFi Direct/AWDL)
+1. **Local Peer Information** - Displays information about the current device including its peer key and cloud connectivity status
+2. **Remote Peers List** - Shows all currently connected remote peers in the mesh network
+3. **Connection Details** - Expandable tiles showing connection types and peer relationships
+4. **Real-time Updates** - Automatically updates as peers join and leave the network
+5. **Cloud Status Indicators** - Visual icons indicating which peers are connected to Ditto Cloud
 
-The implementation uses platform-specific detection to provide accurate status information:
-- **Real Devices**: Attempts to detect actual Bluetooth and WiFi service states
-- **Simulators/Emulators**: Shows "Not available on simulator" with appropriate messaging
-- **Unsupported Platforms**: Shows "Unknown - Check device settings" with settings access
+### Peer Information Displayed
+
+For each peer (local and remote), the view shows:
+- **Device Name** - The human-readable name of the device
+- **Peer Key** - The unique identifier for the peer in the mesh network
+- **Cloud Connectivity** - Icon indicating if the peer is connected to Ditto Cloud (cloud icon) or offline (cloud_off icon)
+- **Active Connections** - Expandable list showing connection details between peers, including connection types
+
+### Real-time Monitoring
+
+The `PeerListView` uses Ditto's presence observer to provide real-time updates:
+- Peers are automatically added when they join the network
+- Peers are removed when they disconnect
+- Connection status changes are reflected immediately
+- No manual refresh required
 
 ### Platform Support
-- ✅ **iOS**: Monitors Bluetooth permissions and detects simulator environments
-- ✅ **Android**: Monitors Bluetooth permissions and detects emulator environments  
-- ✅ **macOS**: Monitors Bluetooth permissions and detects simulator environments
-- ❌ **Web**: Not supported - displays a message indicating web platform limitations
+- ✅ **iOS**: Full support for peer discovery and connection monitoring
+- ✅ **Android**: Full support for peer discovery and connection monitoring  
+- ✅ **macOS**: Full support for peer discovery and connection monitoring
+- ✅ **Linux**: Full support for peer discovery and connection monitoring
+- ❌ **Web**: Limited support - peer-to-peer connections not available on web platform
```

Comment:
It still provides useful info on web, just only about the local peer (e.g. `isConnectedToDittoCloud`). IMO there's no need to list "platform support" for the peer list. On all platforms, it shows details of all peers directly connected.